bibliopixel/__main__.py

```diff
@@ -0,0 +1,43 @@
+import platform
+
+MINIMUM_VERSION = '3.4.0'
+
+if platform.python_version() < MINIMUM_VERSION:
+    raise EnvironmentError(
+        '%s needs Python %s or later: your Python version is %s' % (
+            __package__, MINIMUM_VERSION, platform.python_version()))
+
+try:
+    import argparse
+    import argcomplete
+    import sys
+    from bibliopixel import __description__, __url__
+    from bibliopixel.cli import get_available_commands
+except ImportError:  # pragma: no cover
+    err = sys.exc_info()[1]
+    raise ImportError(str(err))
+
+
+def main():
+    status = 0
+    parser = argparse.ArgumentParser(prog=__package__,
+                                     description=__description__,
+                                     epilog="More info available at: {}".format(__url__))
```

Comment:
We use `'` rather than `"` in BP pretty consistently - just to avoid the extra shift-key.

Reply:
I noticed that, and started keeping with that convention.
I prefer `'` in most cases.
Coming from a C and embedded background, I had some trouble migrating from `"` to `'` for the purposes of matrix/array/string declarations.